integration_tests/cairozero_test.go

```diff
@@ -238,3 +238,23 @@ func memoryRepr(memory []*fp.Element) string {
 	return strings.Join(repr, ", ")
 
 }
+
+func TestFailingRangeCheck(t *testing.T) {
+	compiledOutput, err := compileZeroCode("./failing_contracts/range_check.cairo")
+	require.NoError(t, err)
+
+	_, _, err = runVm(compiledOutput)
+	require.ErrorContains(t, err, "range check builtin failed for value")
+
+	clean("./failing_contracts/")
```

Comment:
This more of a nitpick, but I wouldn't call the folder "failing_contracts" because they are not technically contracts but simple Cairo files. It could be confusing